app/src/organisms/Devices/HistoricalProtocolRun.tsx

```diff
@@ -64,102 +66,96 @@ export function HistoricalProtocolRun(
   return (
     <>
       <Flex
-        justifyContent={JUSTIFY_SPACE_AROUND}
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
         alignItems={ALIGN_CENTER}
         padding={SPACING.spacing8}
         borderTop={BORDERS.lineBorder}
         backgroundColor={
           run.status === 'running' ? COLORS.blue10 : COLORS.white
         }
         width="100%"
+        onClick={() => {
+          setDrawerOpen(!drawer)
+        }}
+        css={css`
+          cursor: pointer;
+        `}
       >
-        <Box
-          onClick={() => {
-            setOffsetDrawerOpen(!offsetDrawerOpen)
-          }}
-          role="button"
-        >
-          <Icon
-            name={offsetDrawerOpen ? 'chevron-up' : 'chevron-down'}
-            width="15px"
-            marginRight={SPACING.spacing8}
-            css={{ cursor: 'pointer' }}
-          />
-        </Box>
-        <StyledText
-          as="p"
-          width="25%"
-          data-testid={`RecentProtocolRuns_Run_${String(protocolKey)}`}
-          onClick={() => {
-            history.push(
-              `${robotName}/protocol-runs/${run.id}/protocolRunDetailsTab?`
-            )
-          }}
-          css={css`
-            cursor: pointer;
-          `}
-          color={COLORS.grey60}
-        >
-          {runDisplayName}
-        </StyledText>
-        {protocolKeyInStoredKeys != null ? (
+        <Flex width="88%" gridGap={SPACING.spacing20}>
+          <StyledText
+            as="p"
+            width="25%"
+            data-testid={`RecentProtocolRuns_Run_${String(protocolKey)}`}
```

Comment:
nit: the String type is something we added awhile back but isn't actually needed? i think we thought it fixed a link warning but it doesnt. So its not a necessity but I try to remove them when i see them in the components I edit

```suggestion
            data-testid={`RecentProtocolRuns_Run_${protocolKey}`}
```

Reply:
As Jethary said, we decided to avoid using `String()` for a case like the above.